config/src/main/resources/examples/filemgr/cmd-line-actions.xml

```diff
@@ -0,0 +1,123 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@chrismattmann I will look into that. Thanks for pointing that out. Didn't know we can do that with a plugin before.